client.go

```diff
@@ -595,6 +609,96 @@ func (c *Client) FollowersOff(channel string) {
 	c.Say(channel, "/followersoff")
 }
 
+// GetVips run twitch command `/vips` with the given channel in argument and returns a string slice of all vips
+func (c *Client) GetVips(channel string) []string {
+	channel = strings.ToLower(channel)
+	ch := make(chan NoticeMessage)
+	makeNew := true
+
+	defer func() {
```

Comment:
i wonder if its possible to hit a race condition on delete after a new one has been created. if the function is called at a very specific time.